### DISCUSSION TITLE
[Q/A][Java SDK (Component)] 服务网格: Proxy模式与Proxyless模式的Dubbo应用，可以互联互通？

### DISCUSSION THREAD

**Discussion by lihuagang03** [apache/dubbo#14118] [Question]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

# 背景
想打通测试环境和本地开发环境的流量路由标签的环境隔离，充分利用稳定的测试环境，避免多套环境和维护成本。

Proxy模式与Proxyless模式的Dubbo应用，可以互联互通。
如官方架构图，Sidecar 与 Proxyless 模式共存

http://static.dubbo.apache.org:8080/zh-cn/overview/core-features/service-mesh/

![image](https://github.com/apache/dubbo/assets/1811851/074684b1-a428-4804-8b29-d47ebd436926)

刘军分享的《基于DubboMesh的标准化云原生微服务体系》和邹毅贤分享的《跨平台的统一控制面实现 Dubbo Mesh》

数据面支持多种部署形态

<img width="1145" alt="image" src="https://github.com/apache/dubbo/assets/1811851/5fa0983e-f883-48b6-9449-ea7750ac254c">

<img width="1194" alt="image" src="https://github.com/apache/dubbo/assets/1811851/bcce4677-4fb0-4d18-9235-c365f4132169">

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
Please submit the discussion in English only:)

**Reply by lihuagang03** [reply]
ok
